pkg/bpf/bpf_suite_test.go

```diff
@@ -19,14 +19,12 @@ func TestBpf(t *testing.T) {
 func checkDataCollected(processesData []ProcessBPFMetrics) {
 	// len > 0
 	Expect(len(processesData)).To(BeNumerically(">", 0))
-	Expect(processesData[0].PID).To(BeNumerically(">", 0))
+	Expect(processesData[0].PID).To(BeNumerically(">=", uint64(0)))
```

Comment:
Should we track pid 0 ?

Reply:
We were always tracking PID 0, not sure why the unit test was only > 0.